d3d/WorkGraphs.md

```diff
@@ -521,9 +521,9 @@ The dispatch grid size can either be part of the input record or be fixed for th
 
 The thread group is fixed in the shader.  
 
-All thread groups that are launched share the same set of input parameters.  The exception is the ususal ID system values which identify individual threads within the set.

```

Comment:
Actually it is the full set, as there's system values that tell you where you are in the overall grid.   Could say "group/grid" instead of set.